src/components/VersionSelect/index.tsx

```diff
@@ -64,24 +67,31 @@ class VersionSelectBase extends React.Component<PublicProps> {
 
         <Form.Group as={Col} className={className}>
           <Form.Label>{label}</Form.Label>
-          <Form.Control as="select" value={value} onChange={this.onChange}>
-            {listedVersions.length && (
-              <optgroup
-                className={styles.listedGroup}
-                label={gettext('Listed')}
-              >
-                {listedVersions.map(this.renderOption)}
-              </optgroup>
-            )}
-            {unlistedVersions.length && (
-              <optgroup
-                className={styles.unlistedGroup}
-                label={gettext('Unlisted')}
-              >
-                {unlistedVersions.map(this.renderOption)}
-              </optgroup>
-            )}
-          </Form.Control>
+
+          {isLoading ? (
+            <div className="form-control">
```

Comment:
Could you make it clear somehow that `form-control` is a hardcoded value that must match that of `<Form.Control>`? If possible, maybe you could control the class name? I think there is at least an option to specify a class name prefix.

Reply:
unfortunately, we have to use `.form-control`. It comes from the Bootstrap CSS package. We could use `@extend .form-control` in our own CSS class definition, but that would be very similar.